Extensions/Steamworks/JsExtension.js

```diff
@@ -0,0 +1,1250 @@
+        _('Obtains the Steam ID of the user that owns a lobby.'),
+        _('Matchmaking'),
+        'JsPlatform/Extensions/steam.svg'
+      )
+      .addParameter('string', 'The ID of the lobby', '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Steamworks/steamworkstools.js')
+      .setFunctionName('gdjs.steamworks.getLobbyOwner');
+
+    extension
+      .addCondition(
+        'PlayerOwnsApp',
+        _('Player owns an application'),
+        _('Checks if the current user owns an application on Steam.'),
+        _('App _PARAM0_ owned on Steam'),
+        _('Steam Apps'),
+        'JsPlatform/Extensions/steam.svg',
+        'JsPlatform/Extensions/steam.svg'
+      )
+      .addParameter(
+        'identifier',
+        'The Steam App ID of the application',
+        'SteamAppID',
+        false
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Steamworks/steamworkstools.js')
+      .setFunctionName('gdjs.steamworks.isAppOwned');
+
+    extension
+      .addCondition(
+        'PlayerInstalledApp',
+        _('Player installed an application'),
+        _(
+          'Checks if the current user has a Steam application currently installed.'
+        ),
+        _('App _PARAM0_ installed from Steam'),
+        _('Steam Apps'),
+        'JsPlatform/Extensions/steam.svg',
+        'JsPlatform/Extensions/steam.svg'
+      )
+      .addParameter(
+        'identifier',
+        'The Steam App ID of the application',
+        'SteamAppID',
+        false
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Steamworks/steamworkstools.js')
+      .setFunctionName('gdjs.steamworks.isAppInstalled');
+
+    extension
+      .addCondition(
+        'PlayerInstalledDLC',
+        _('Player installed DLC'),
+        _('Checks if the current user has installed a piece of DLC.'),
+        _('DLC _PARAM0_ installed from Steam'),
+        _('Steam Apps'),
+        'JsPlatform/Extensions/steam.svg',
+        'JsPlatform/Extensions/steam.svg'
+      )
+      .addParameter(
+        'identifier',
+        'The Steam App ID of the DLC',
+        'SteamAppID',
+        false
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Steamworks/steamworkstools.js')
+      .setFunctionName('gdjs.steamworks.isDLCInstalled');
+
+    extension
+      .addStrExpression(
+        'InstalledApplicationPath',
+        _('Get installed app path'),
+        _('Gets the path to an installed Steam application.'),
+        _('Steam Apps'),
+        'JsPlatform/Extensions/steam.svg'
+      )
+      .addParameter(
+        'identifier',
+        'The Steam App ID of the application',
+        'SteamAppID',
+        false
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Steamworks/steamworkstools.js')
+      .setFunctionName('gdjs.steamworks.getAppInstallDirectory');
+
+    extension
+      .addCondition(
+        'PlayerIsVACBanned',
```

Comment:
nit:
I know this can be a lot of renaming, but in theory actions/conditions must follow the same rules as a function for naming:

Conditions: Is/HasSomethingBeingDone. Here: `IsPlayerVACBanner`.
Actions: [Verb]Something. (for example: "BanPlayerFromXxx")

(Only exception is expressions for which the verb "Get" is implied and never added).